test/panicround-test.js

```diff
@@ -0,0 +1,10 @@
+import chai from 'chai';
+const expect = chai.expect;
+
+describe('PanicRound', () => {
+  it('should be a function', function() {
+  expect(PanicRound).to.be.a('function');
+});
+})
+
+export default PanicRound;
```

Comment:
We don't need to export the panicRound test.  We would want to import the panicRound class.